components/org.wso2.carbon.identity.api.server.claim.management/org.wso2.carbon.identity.rest.api.server.claim.management.v1/src/main/java/org/wso2/carbon/identity/rest/api/server/claim/management/v1/core/ServerClaimManagementService.java

```diff
@@ -443,7 +443,6 @@ public List<LocalClaimResDTO> getLocalClaims(Boolean excludeIdentityClaims, Stri
     public void updateLocalClaim(String claimId, LocalClaimReqDTO localClaimReqDTO) {
 
         try {
-            validateClaimModificationEligibility();
```

Comment:
Isn't this allow to update any property of the claim in sub org?

Reply:
updated to only allow claim mapping updates with [b649520](https://github.com/wso2/identity-api-server/pull/771/commits/b649520e364d1f279c32855254a59ac97dc680e0)